cli/cli-interpreter/src/main/java/org/infinispan/cli/interpreter/Interpreter.java

```diff
@@ -45,25 +45,20 @@
    private static final long DEFAULT_SESSION_REAPER_WAKEUP_INTERVAL = 60000l; // in millis
    private static final long DEFAULT_SESSION_TIMEOUT = 360000l; // in millis
 
-   private EmbeddedCacheManager cacheManager;
+   @Inject private EmbeddedCacheManager cacheManager;
+   @Inject private TimeService timeService;
```

Comment:
Don't know how I missed this before, but do we really want to put the annotation on the same line as the field?

Reply:
I like the declaration to be concise, since `@Inject` does not have any parameters. 8 characters including space don't justify newline.

I actually put the annotation on separate line when component name is set through another annotation, but not in the default case.